server/src/leihs/procurement/resources/image.clj

```diff
@@ -18,7 +17,7 @@
 (defn image-query
   [id]
   (-> image-base-query
-      (sql/where [:= :procurement_images.id id])))
+      (sql/where [:= :procurement_images.id (to-uuid id)])))
```

Comment:
`to-uuid` is fine, but i am wondering if we don't have a schema parser for that...?

Reply:
oh ok. i think the schema is not involved here. then that's fine.